donkeycar/management/tub_gui.py

```diff
@@ -0,0 +1,598 @@
+#!/usr/bin/env python3
+import math
+import os
+from collections import namedtuple
+import tkinter as tk
+from tkinter import ttk, filedialog
+import time
+from threading import Thread, active_count
+from PIL import ImageTk, Image
+import pandas as pd
+import yaml
```

Comment:
Why YAML :disappointed:  ?

Maybe try using `json` consistently ?

Reply:
This is used only for storing the config file of the app. Json is a bit inconvenient to edit with all the `=` and `{`, etc,  and you might need to modify this file if you want a graphical progress bar for unknown data. So adding a row into this file is very easy. Compare it to a `.condarc` or similar file.